patches/api/0496-API-to-allow-disallow-tick-sleeping.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Abel <abelvanhulst@gmail.com>
+Date: Tue, 12 Nov 2024 22:25:35 +0100
+Subject: [PATCH] API to allow/disallow tick sleeping
+
+
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index 0b78564256ebc647ebac402e549d86ab6e307c8d..f709429e258e487d75cf6fe3d2ee88d004890345 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -2572,4 +2572,16 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
+      */
+     boolean isOwnedByCurrentRegion(@NotNull Entity entity);
+     // Paper end - Folia region threading API
++
++    // Paper start - API to allow/disallow tick sleeping
++    /**
++     * Allow or disallow the server to sleep/pause.This only effects the next time the server is paused. If the server is currently paused, it will only go out of sleep when someone joins.
```

Comment:
```suggestion
+     * Allow or disallow the server to sleep/pause. This only affects the next time the server is paused. If the server is currently paused, it will only go out of sleep when someone joins.
```
Is that last sentence still true?

Reply:
Don't know, will test in a bit